staging/src/k8s.io/kubectl/pkg/cmd/create/create_job.go

```diff
@@ -195,16 +195,27 @@ func (o *CreateJobOptions) Run() error {
 			return fmt.Errorf("from must be an existing cronjob")
 		}
 
-		uncastVersionedObj, err := scheme.Scheme.ConvertToVersion(infos[0].Object, batchv1beta1.SchemeGroupVersion)
+		uncastVersionedObj, err := scheme.Scheme.ConvertToVersion(infos[0].Object, batchv1.SchemeGroupVersion)
```

Comment:
I'm surprised kubectl is still doing conversion... I thought this got dropped in 1.18. Is this effectively just checking the apiVersion/kind and decoding directly to the versioned type?

Reply:
Yeah, I think that was my bad from the past. I used unstructured and needed to convert. I fixed that to use typed which is much cleaner now.